graph/appender.go

```diff
@@ -12,10 +13,12 @@ type AppenderVendorInfo map[string]interface{}
 // can re-use the information.  A new instance is generated for graph and
 // is initially empty.
 type AppenderGlobalInfo struct {
-	Business    *business.Layer
-	HomeCluster string
-	PromClient  *prometheus.Client
-	Vendor      AppenderVendorInfo // telemetry vendor's global info
+	Business         *business.Layer
+	HomeCluster      string
+	GatewayCrds      models.Gateways
+	IngressWorkloads map[string][]models.WorkloadListItem // Map key is the namespace name. Value is the list of ingress workloads in that namespace.
```

Comment:
I'm quite sure, because `NamespaceInfo` is different per call to the appender. But yes, this is only for "caching", so if the Kiali cache is guaranteed, I can rely on it and re-fetch data on every call to remove these calls. I'll do it.

Reply:
I think it's OK to rely on the cache, if the cache data is in any way manipulated repeatedly then the graph/telemetry/istio/appender/appender.go approach analogous to getWorkload/getWorkloadList could be good.